functests/4_latency/latency.go

```diff
@@ -347,10 +347,10 @@ func extractLatencyValues(testName string, exp string, node *corev1.Node) string
 	maximumRegex, err := regexp.Compile(exp)
 	Expect(err).ToNot(HaveOccurred())
 
-	latencies := maximumRegex.FindStringSubmatch(out)[1]
-	Expect(maximumLatency).ToNot(BeNil())
+	latencies := maximumRegex.FindStringSubmatch(out)
+	Expect(len(latencies)).To(Equal(2))
```

Comment:
why 2? should not be like 16 (17?) in the example above?

Reply:
Yeah, this should probably use the same value the test pod used, to be future proof

Reply:
Oh, I think I know why. The regex has two groups. The second one contains the list of the results.

Reply:
in general, it returns two outputs, the first one is the matched string and the second is sub-group, it looks like
```
["Maximum:       23 32 23 42 19 20 29 19 23 33 11 (us)",  "23 32 23 42 19 20 29 19 23 33 11 "]
```

Reply:
right, my fault

Reply:
Will this change apply to hwlat and cyclic as well?

Reply:
I do not change the logic under the method, I just want to guarantee that we will not fail with the index error once you try to get the right string under `maximumRegex.FindStringSubmatch(out)[1]`